src/main/java/io/reactivex/Completable.java

```diff
@@ -260,6 +260,18 @@ public static Completable fromCallable(final Callable<?> callable) {
         return new CompletableFromFlowable<T>(flowable);
     }
     
+    @SchedulerSupport(SchedulerSupport.NONE)
```

Comment:
Looks awkward because it fact it's supported through `subscribeOn`, maybe rename to `@DefaultScheduler`?